### DISCUSSION TITLE
Documentation as pdf

### DISCUSSION THREAD

**Discussion by TEMHITHORPHE** [django-commons/django-unicorn#104] [Ideas]
** Don't know if this is the right place for this though, opening a PR didn't seem right … 

For many reading this, stable/regular internet access is probably not an issue, but for some of us folks in developing nations, it might be quite the challenge, so having an offline PDF documentation is a huge plus, that way I can work and learn while offline! … the only remedy I plan to apply now is to download the entire site offline using HTTrack, which as you can guess, is not all that comfy. 

Ladies and gentlemen, his royal-developer adamghill, open-source devotees, with theses few points of mine … I hope I have been able to cement my point and convince the majority that "WE DO NEED A LINK TO THE DOCUMENTATION IN PDF FORMAT!" right there on the documentation page! … it will sure be helpful

**Comment by adamghill** [comment]
I love this idea! I've actually been spending the past day re-writing all of the documentation into a format that `sphinx` can handle. One benefit (other than the tedious hard-coded HTML I deal with now) is that I should be able to export a version of the documentation into a PDF automatically.

I'll try to finish the conversion as fast as I can and will post a link to the PDF in this discussion thread.

**Reply by adamghill** [reply]
In case you want to follow along, https://github.com/adamghill/django-unicorn.com/pull/4 is the PR for converting the documentation to `sphinx`.

**Reply by TEMHITHORPHE** [reply]
Thanks! … this will! help! … appreciate the work you putting in!

**Reply by adamghill** [reply]
This is a pretty rough first draft because I haven't found a nicer PDF template, but all of the information should at least be in here. [unicorn-0.13.0.pdf](https://github.com/adamghill/django-unicorn/files/5756472/unicorn-0.13.0.pdf)

I'll try to make this look a little better and then put a link on the site.

**Reply by adamghill** [reply]
I added a link to that pdf to the sidebar on the https://www.django-unicorn.com/docs/. I also made an issue (https://github.com/adamghill/django-unicorn.com/issues/12) to make the PDF look a little better.
